src/features/auth/components/PageLayout/index.js

```diff
@@ -3,23 +3,45 @@ import PropTypes from 'prop-types';
 import { Trans } from 'react-i18next';
 import { Link } from 'react-router-dom';
 
-import { Layout, Header, Footer, Menu } from '../../../../common/components';
+import { Footer, Header, Icon, Layout, Menu, Popover } from '../../../../common/components';
 
 const PageLayout = ({ children }) => (
   <Layout>
     <Header>
-      {() => [
-        <Menu.Item key="signup">
-          <Link to="/sign-up">
-            <Trans i18nKey="nav.signup">Sign Up</Trans>
-          </Link>
-        </Menu.Item>,
-        <Menu.Item key="login">
-          <Link to="/login">
-            <Trans i18nKey="nav.login">Log In</Trans>
-          </Link>
-        </Menu.Item>,
-      ]}
+      {({ activePath, hideMenu, showMenu, isMenuVisible, isMobile }) => {
```

Comment:
props from here are coming from `ResponsiveNavMenu ` but its not visible here. looks confusing doesnt it?